docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1422,6 +1422,21 @@
                     </td>
                 </tr>
 
+                <tr>
+                    <td class="property" id="stores-rocksdb-log-file-size">stores.<span class="store">store-name</span>.<br>rocksdb.max.log.file.size.bytes</td>
+                    <td class="default">67108864</td>
+                    <td class="description">
+                        The maximum size in bytes of the RocksDB LOG file before it is rotated.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="stores-rocksdb-num-log-files">stores.<span class="store">store-name</span>.<br>rocksdb.keep.log.file.num</td>
```

Comment:
minor: What do you think about renaming this property to `num.log.files`? A nice thing is that it's arguably consistent with the other Samza property `num.write.buffers`?  

(Or, is the goal to mirror rocksDb config keys?)